src/_pytest/terminal.py

```diff
@@ -357,6 +357,9 @@ def rewrite(self, line, **markup):
 
     def write_sep(self, sep, title=None, **markup):
         self.ensure_newline()
+        if sep == "_":
```

Comment:
This seems unrelated?

Reply:
Fixed.  That's what I want to use it for, but it needs an adjusted TerminalWriter, of course.

I wonder if I should import it via `_pytest.compat` here already, so that there's really a single place where to extend it, and/or put it into `_pytest._io`?